pkg/nat/nat.go

```diff
@@ -84,6 +102,14 @@ func (p Port) Int() int {
 	return int(port)
 }
 
+// Range returns the start/end port numbers of a Port range as ints
+func (p Port) Range() (int, int) {
+	// We don't need to check for an error because we're going to
+	// assume that any error would have been found, and reported, in NewPort()
```

Comment:
Is there a reason why we are trying to ignore the error (other than reason mentioned here?).
I guess we could break this assumption with a simple Unit-Test.
Though it is not something we will see today, this assumption can lead to issues down the line.
Hence it would be better to not ignore the error.